lib/xenopsd.ml

```diff
@@ -175,13 +185,18 @@ let options =
     , "True if IOMMU contexts of PCI devices are needed to be placed in \
        quarantine" )
   ; ( "vm-guest-agent-xenstore-quota"
-    , Arg.Set_int vm_guest_agent_xenstore_quota
-    , (fun () -> string_of_int !vm_guest_agent_xenstore_quota)
-    , "Maximum entries in VM xenstore trees watched by xenopsd" )
+    , Arg.String (fun s ->
+          vm_guest_agent_xenstore_quota_bytes := max_bytes_of_entries (int_of_string s))
```

Comment:
`int_of_string` can fail - but it would be obvious

Reply:
Yes, it would fail right on startup